TileStache/Goodies/VecTiles/transform.py

```diff
@@ -3093,3 +2972,155 @@ def normalize_medical_kind(shape, properties, fid, zoom):
                 properties['specialty'] = specialty.split(';')
 
     return (shape, properties, fid)
+
+
+class _AnyMatcher(object):
+    def match(self, other):
+        return True
+
+    def __repr__(self):
+        return "*"
+
+
+class _NoneMatcher(object):
+    def match(self, other):
+        return other is None
+
+    def __repr__(self):
+        return "-"
+
+
+class _SomeMatcher(object):
+    def match(self, other):
+        return other is not None
+
+    def __repr__(self):
+        return "+"
+
+
+class _ExactMatcher(object):
+    def __init__(self, value):
+        self.value = value
+
+    def match(self, other):
+        return other == self.value
+
+    def __repr__(self):
+        return repr(self.value)
+
+
+class _SetMatcher(object):
+    def __init__(self, values):
+        self.values = values
+
+    def match(self, other):
+        return other in self.values
+
+    def __repr__(self):
+        return repr(self.value)
+
```

Comment:
We could transition the `match` function to be `__call__`, but we can do that later if we wanted.